tonytanaka/index.js

```diff
@@ -5,11 +5,15 @@ let x = Math.floor(Math.random() * count) + 1;
 console.log(chrs[x]);
 
 // exercise 2
-let sy = "%&    § !* @= ";
-let cleanSy = sy.replace(/\s/g, "");
+//let sy = "%&    § !* @= ";
+let sy = " %&§!*@= ";
+//let cleanSy = sy.replace(/\s/g, "");
 /* \s is the regex for "whitespace", and g is the 
 "global" flag, meaning match ALL \s (whitespaces).*/
+let cleanSy = sy.trim();
+console.log(cleanSy);
 let countSy = cleanSy.length;
+console.log(countSy);
 let y = Math.floor(Math.random() * countSy) + 1;
```

Comment:
Hey Tony, in line 17 you can delete +1 because when i try to run the code I get undefined after every 4-5 tries.

Reply:
you`re right! 7 is the lenght, but the index goes up to 6

Reply:
👍🏻